### PR TITLE
Add Remomix.app v0.2.1

### PR DESCRIPTION
**Pull request by johndbritton**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/remomix.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask 'remomix' do
+  version '0.2.1'
+  sha256 '45a2798544faeea741d43833f4352178ad0c4fe56213a4a23f31818b39ad5c0d'
+
+  # github.com/johndbritton was verified as official when first introduced to the cask
+  url 'https://github.com/johndbritton/remomix-releases/releases/download/v0.2.1/Remomix-0.2.1-osx.zip'
```
Comment (adidalal) [inline Casks/remomix.rb]: Also, mind adding an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

Comment (adidalal) [inline Casks/remomix.rb]: Use `#{version}` instead of hardcoding in `0.2.1`

File: Casks/remomix.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  name 'Remomix'
+  homepage 'http://remomix.com'
```
Comment (adidalal) [inline Casks/remomix.rb]: Currently a "Heroku - app not found" page, FYI

File: Casks/remomix.rb (hunk 1 above)
Comment (johndbritton) [inline Casks/remomix.rb]: There's no appcast URL for this application. It's using the [Electron auto-updater](https://github.com/electron/electron/blob/master/docs/api/auto-updater.md) which does not use the Appcast format.

File: Casks/remomix.rb (hunk 2 above)
Comment (johndbritton) [inline Casks/remomix.rb]: This is expected.

File: Casks/remomix.rb (hunk 1 above)
Comment (johndbritton) [inline Casks/remomix.rb]: Didn't realize I got a free appcast from GitHub Releases, added and rebased.

Comment (adidalal) [inline Casks/remomix.rb]: The auto-update URL is fine. We will use it to facilitate Cask auto-updates at some point in the future.

Also, `https://github.com/johndbritton/remomix-releases/releases.atom` is fine too, we just want a URL that will change its checksum when a new release comes out.

File: Casks/remomix.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+cask 'remomix' do
+  version '0.2.1'
+  sha256 '45a2798544faeea741d43833f4352178ad0c4fe56213a4a23f31818b39ad5c0d'
+
+  # github.com/johndbritton was verified as official when first introduced to the cask
```
Comment (jawshooah) [inline Casks/remomix.rb]: Should contain the entire repo slug: `github.com/johndbritton/remomix-releases`

File: Casks/remomix.rb (hunk 2 above)
Comment (adidalal) [inline Casks/remomix.rb]: Is there any other source of verification that `github.com/johndbritton/remomix-releases` and `remomix.com` are the same app/the link given is the correct one?

File: Casks/remomix.rb [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+  url "https://github.com/johndbritton/remomix-releases/releases/download/v#{{version}}/Remomix-#{{version}}-osx.zip"
```
Comment (adidalal) [inline Casks/remomix.rb]: Just single {, ie. `#{version}`

Comment (jawshooah) [inline Casks/remomix.rb]: You've got [some extra brackets](https://travis-ci.org/caskroom/homebrew-cask/jobs/145396185#L384-L391) here. Just replace `#{{version}}` with `#{version}`

**Comment by jawshooah** [comment]
Thank you for the contribution. It needed some fixes, so they were made in f8e1f004453c511a328ccafa2fb5c72e0d6f03c5. Your contribution is still included (and still credited to you), with the appropriate modifications. Please feel free to ask about any of the changes.

**Pull request closed without merging**